Include ordering is broken

Some (I have not produced an exhaustive list) of the convenience `*.mk` includes are relying on variables at stages where they don't exist yet. Example repro:

```
[efried@efried ~]$ cd $(mktemp -d)
[efried@efried tmp.9ZAD5Q9Qzz]$ echo 'include /home/efried/go/src/github.com/openshift/build-machinery-go/make/targets/openshift/controller-gen.mk' > Makefile
[efried@efried tmp.9ZAD5Q9Qzz]$ make ensure-controller-gen 
Installing controller-gen into '_output/tools/bin/controller-gen'
mkdir -p '/bin/'
curl -s -f -L https://github.com/openshift/kubernetes-sigs-controller-tools/releases/download/v0.6.0/controller-gen-linux-amd64 -o '_output/tools/bin/controller-gen'
make: *** [/home/efried/go/src/github.com/openshift/build-machinery-go/make/targets/openshift/controller-gen.mk:11: ensure-controller-gen] Error 23
```

Here's what's happening:
1. controller-gen.mk is [computing `controller_gen_dir`](https://github.com/openshift/build-machinery-go/blob/792d61687197876958897f1b215aada85f6865e9/make/targets/openshift/controller-gen.mk#L5) with `:=` -- which means it happens just once -- during the first pass. Since `CONTROLLER_GEN` is defined with `=`, it is computed dynamically at this time, based on `PERMANENT_TMP_GOPATH`, which is not yet defined because we haven't [included tmp.mk](https://github.com/openshift/build-machinery-go/blob/792d61687197876958897f1b215aada85f6865e9/make/targets/openshift/controller-gen.mk#L30) yet. So `controller_gen_dir` gets (permanently) assigned the value `/bin/` here.
1. Continuing the first pass, we [use the existence](https://github.com/openshift/build-machinery-go/blob/792d61687197876958897f1b215aada85f6865e9/make/targets/openshift/controller-gen.mk#L8) of whatever's at `$(CONTROLLER_GEN)` to determine what the contents of the `ensure-controller-gen` rule should be. We again dynamically compute `CONTROLLER_GEN`, but `PERMANENT_TMP_GOPATH` is still undefined, so `CONTROLLER_GEN` gets the value `/bin/controller-gen`. That (presumably) doesn't exist, so we lay down the [`if` version of the rule](https://github.com/openshift/build-machinery-go/blob/792d61687197876958897f1b215aada85f6865e9/make/targets/openshift/controller-gen.mk#L9-L12).
1. Continuing the first pass, we finally include `tmp.mk` and the `PERMANENT_TMP_GOPATH` variable is (permanently, because `:=`) [assigned](https://github.com/openshift/build-machinery-go/blob/792d61687197876958897f1b215aada85f6865e9/make/lib/tmp.mk#L2).
1. I run `make ensure-controller-gen`.
1. The `info` message computes `CONTROLLER_GEN` dynamically again, but now since `PERMANENT_TMP_GOPATH` is defined, it actually looks correct.
1. The `mkdir -p` "succeeds" (at least on my machine) because `/bin` already exists (symlink to `/usr/bin`).
1. The [`curl` command](https://github.com/openshift/build-machinery-go/blob/792d61687197876958897f1b215aada85f6865e9/make/targets/openshift/controller-gen.mk#L11) again computes the `CONTROLLER_GEN` variable properly -- but since the `mkdir -p` didn't actually create `_output/tools/bin`, it fails.

I can fix this partway by changing the assignment of `controller_gen_dir` to be dynamic:

```diff
diff --git a/make/targets/openshift/controller-gen.mk b/make/targets/openshift/controller-gen.mk
index 163603c..44efaf3 100644
--- a/make/targets/openshift/controller-gen.mk
+++ b/make/targets/openshift/controller-gen.mk
@@ -2,7 +2,7 @@ self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
 
 CONTROLLER_GEN_VERSION ?=v0.6.0
 CONTROLLER_GEN ?=$(PERMANENT_TMP_GOPATH)/bin/controller-gen
-controller_gen_dir :=$(dir $(CONTROLLER_GEN))
+controller_gen_dir =$(dir $(CONTROLLER_GEN))
 
 ensure-controller-gen:
 ifeq "" "$(wildcard $(CONTROLLER_GEN))"
```

Now when I try:

```
[efried@efried tmp.9ZAD5Q9Qzz]$ make ensure-controller-gen 
Installing controller-gen into '_output/tools/bin/controller-gen'
mkdir -p '_output/tools/bin/'
curl -s -f -L https://github.com/openshift/kubernetes-sigs-controller-tools/releases/download/v0.6.0/controller-gen-linux-amd64 -o '_output/tools/bin/controller-gen'
chmod +x '_output/tools/bin/controller-gen';
```

Great! Now let's just check that it's idempotent as intended:

```
[efried@efried tmp.9ZAD5Q9Qzz]$ make ensure-controller-gen 
Installing controller-gen into '_output/tools/bin/controller-gen'
mkdir -p '_output/tools/bin/'
curl -s -f -L https://github.com/openshift/kubernetes-sigs-controller-tools/releases/download/v0.6.0/controller-gen-linux-amd64 -o '_output/tools/bin/controller-gen'
chmod +x '_output/tools/bin/controller-gen';
```

Nope, it's laying down the file again. Referring back to bullet 2 above, it's because `$(wildcard $(CONTROLLER_GEN))` is (still) resolving to `/bin/controller-gen`, which doesn't exist; so we're still laying down the first version of the rule.

We could fix *both* problems if we `include tmp.mk` *before* `PERMANENT_TMP_GOPATH` is needed -- i.e. at the top of the file. Based on the comment above that `include`, the only reason it's at the end is so we can rely on `self_dir` having the expected value throughout the file up to this point. But:
- Except for generating these `include`s everywhere, `self_dir` is only used in four places [1](https://github.com/openshift/build-machinery-go/blob/792d61687197876958897f1b215aada85f6865e9/make/targets/openshift/deps-glide.mk#L2), [2, 3](https://github.com/openshift/build-machinery-go/blob/792d61687197876958897f1b215aada85f6865e9/make/targets/openshift/deps.mk#L3-L4), [4](https://github.com/openshift/build-machinery-go/blob/master/make/targets/openshift/operator/telepresence.mk#L2).
- `self_dir` is patently *not* part of the "API", since there's no telling what its value might be by the time I try to use it in my root Makefile.

So IMHO it would be cleaner all the way around to do the includes up top and avoid reusing the variable name. In most places, we could just inline the computation, e.g.

```diff
diff --git a/make/targets/openshift/controller-gen.mk b/make/targets/openshift/controller-gen.mk
index 163603c..03b962e 100644
--- a/make/targets/openshift/controller-gen.mk
+++ b/make/targets/openshift/controller-gen.mk
@@ -1,4 +1,7 @@
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+       ../../lib/golang.mk \
+       ../../lib/tmp.mk \
+)
 
 CONTROLLER_GEN_VERSION ?=v0.6.0
 CONTROLLER_GEN ?=$(PERMANENT_TMP_GOPATH)/bin/controller-gen
@@ -22,10 +25,3 @@ clean-controller-gen:
 
 clean: clean-controller-gen
 
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
-       ../../lib/golang.mk \
-       ../../lib/tmp.mk \
-)
```

And in those few places where we actually need the variable, we can use one with a unique name so we don't need to worry about conflicts, e.g.

```diff
diff --git a/make/targets/openshift/deps.mk b/make/targets/openshift/deps.mk
index 956698c..37a666f 100644
--- a/make/targets/openshift/deps.mk
+++ b/make/targets/openshift/deps.mk
@@ -1,8 +1,8 @@
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+_self_dir_openshift_deps :=$(dir $(lastword $(MAKEFILE_LIST)))
 
-deps_gomod_mkfile := $(self_dir)/deps-gomod.mk
-deps_glide_mkfile := $(self_dir)/deps-glide.mk
-include $(addprefix $(self_dir), \
+deps_gomod_mkfile := $(_self_dir_openshift_deps)/deps-gomod.mk
+deps_glide_mkfile := $(_self_dir_openshift_deps)/deps-glide.mk
+include $(addprefix $(_self_dir_openshift_deps), \
        ../../lib/golang.mk \
 )
 
```

--- a/make/default.mk
+++ b/make/default.mk
@@ -1,4 +1,10 @@
-self_dir := $(dir $(lastword $(MAKEFILE_LIST)))
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	targets/openshift/deps.mk \
+	targets/openshift/images.mk \
+	targets/openshift/bindata.mk \
+	targets/openshift/codegen.mk \
+	golang.mk \
+)
 
 # We extend the default verify/update for Golang
 
@@ -9,15 +15,3 @@ verify: verify-bindata
 update: update-codegen
 update: update-bindata
 .PHONY: update
-
-
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
-	targets/openshift/deps.mk \
-	targets/openshift/images.mk \
-	targets/openshift/bindata.mk \
-	targets/openshift/codegen.mk \
-	golang.mk \
-)

--- a/make/golang.mk
+++ b/make/golang.mk
@@ -1,8 +1,10 @@
 all: build
 .PHONY: all
 
-self_dir := $(dir $(lastword $(MAKEFILE_LIST)))
-
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	targets/help.mk \
+	targets/golang/*.mk \
+)
 
 verify: verify-gofmt
 verify: verify-govet
@@ -18,12 +20,3 @@ test: test-unit
 
 clean: clean-binaries
 .PHONY: clean
-
-
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
-	targets/help.mk \
-	targets/golang/*.mk \
-)

--- a/make/operator.mk
+++ b/make/operator.mk
@@ -1,11 +1,4 @@
-self_dir := $(dir $(lastword $(MAKEFILE_LIST)))
-
-
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
 	default.mk \
 	targets/openshift/operator/*.mk \
 )
-

--- a/make/targets/golang/build.mk
+++ b/make/targets/golang/build.mk
@@ -1,4 +1,6 @@
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	../../lib/golang.mk \
+)
 
 define build-package
 	$(if $(GO_BUILD_BINDIR),mkdir -p '$(GO_BUILD_BINDIR)',)
@@ -20,10 +22,3 @@ clean-binaries:
 
 clean: clean-binaries
 .PHONY: clean
-
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
-	../../lib/golang.mk \
-)

--- a/make/targets/golang/test-unit.mk
+++ b/make/targets/golang/test-unit.mk
@@ -1,4 +1,6 @@
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	../../lib/golang.mk \
+)
 
 test-unit:
 ifndef JUNITFILE
@@ -10,10 +12,3 @@ endif
 	set -o pipefail; $(GO) test $(GO_MOD_FLAGS) $(GO_TEST_FLAGS) -json $(GO_TEST_PACKAGES) | gotest2junit > $(JUNITFILE)
 endif
 .PHONY: test-unit
-
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
-	../../lib/golang.mk \
-)

--- a/make/targets/golang/verify-update.mk
+++ b/make/targets/golang/verify-update.mk
@@ -1,4 +1,6 @@
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	../../lib/golang.mk \
+)
 
 go_files_count :=$(words $(GO_FILES))
 
@@ -27,10 +29,3 @@ verify-govet:
 verify-golint:
 	$(GOLINT) $(GO_PACKAGES)
 .PHONY: verify-govet
-
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
-	../../lib/golang.mk \
-)

--- a/make/targets/golang/version.mk
+++ b/make/targets/golang/version.mk
@@ -1,4 +1,7 @@
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	../../lib/golang.mk \
+	../../lib/tmp.mk \
+)
 
 .empty-golang-versions-files:
 	@rm -f "$(PERMANENT_TMP)/golang-versions" "$(PERMANENT_TMP)/named-golang-versions"
@@ -53,13 +56,3 @@ $(if $(1),$(call verify-Dockerfile-builder-golang-version,$(1))) \
 $(if $(wildcard ./.ci-operator.yaml),$(if $(shell grep 'build_root_image:' .ci-operator.yaml 2>/dev/null),$(call verify-buildroot-golang-version))) \
 $(if $(wildcard ./go.mod),$(call verify-go-mod-golang-version))
 endef
-
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
-	../../lib/golang.mk \
-)
-include $(addprefix $(self_dir), \
-	../../lib/tmp.mk \
-)

--- a/make/targets/openshift/controller-gen.mk
+++ b/make/targets/openshift/controller-gen.mk
@@ -1,4 +1,7 @@
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	../../lib/golang.mk \
+	../../lib/tmp.mk \
+)
 
 CONTROLLER_GEN_VERSION ?=v0.6.0
 CONTROLLER_GEN ?=$(PERMANENT_TMP_GOPATH)/bin/controller-gen
@@ -21,11 +24,3 @@ clean-controller-gen:
 .PHONY: clean-controller-gen
 
 clean: clean-controller-gen
-
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
-	../../lib/golang.mk \
-	../../lib/tmp.mk \
-)

--- a/make/targets/openshift/crd-schema-gen.mk
+++ b/make/targets/openshift/crd-schema-gen.mk
@@ -1,4 +1,10 @@
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	../../lib/golang.mk \
+	../../lib/tmp.mk \
+	../../targets/openshift/controller-gen.mk \
+	../../targets/openshift/yq.mk \
+	../../targets/openshift/yaml-patch.mk \
+)
 
 # $1 - crd file
 # $2 - patch file
@@ -76,15 +82,3 @@ verify: verify-generated
 define add-crd-gen
 $(eval $(call add-crd-gen-internal,$(1),$(2),$(3),$(4)))
 endef
-
-
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
-	../../lib/golang.mk \
-	../../lib/tmp.mk \
-	../../targets/openshift/controller-gen.mk \
-	../../targets/openshift/yq.mk \
-    ../../targets/openshift/yaml-patch.mk \
-)

--- a/make/targets/openshift/deps-glide.mk
+++ b/make/targets/openshift/deps-glide.mk
@@ -1,5 +1,4 @@
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
-scripts_dir :=$(self_dir)/../../../scripts
+scripts_dir :=$(dir $(lastword $(MAKEFILE_LIST)))/../../../scripts
 
 # We need to force localle so different envs sort files the same way for recursive traversals
 deps_diff :=LC_COLLATE=C diff --no-dereference -N

--- a/make/targets/openshift/deps-gomod.mk
+++ b/make/targets/openshift/deps-gomod.mk
@@ -1,4 +1,6 @@
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	../../lib/golang.mk \
+)
 
 # We need to force localle so different envs sort files the same way for recursive traversals
 deps_diff :=LC_COLLATE=C diff --no-dereference -N
@@ -32,11 +34,3 @@ update-deps-overrides:
 	$(call restore-deps,$(tmp_dir))
 	cp "$(tmp_dir)"/{updated,current}/deps.diff
 .PHONY: update-deps-overrides
-
-
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
-	../../lib/golang.mk \
-)

--- a/make/targets/openshift/deps.mk
+++ b/make/targets/openshift/deps.mk
@@ -1,8 +1,10 @@
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+# Use a unique variable name to avoid conflicting with generic
+# `self_dir` elsewhere.
+_self_dir_openshift_deps :=$(dir $(lastword $(MAKEFILE_LIST)))
 
-deps_gomod_mkfile := $(self_dir)/deps-gomod.mk
-deps_glide_mkfile := $(self_dir)/deps-glide.mk
-include $(addprefix $(self_dir), \
+deps_gomod_mkfile := $(_self_dir_openshift_deps)/deps-gomod.mk
+deps_glide_mkfile := $(_self_dir_openshift_deps)/deps-glide.mk
+include $(addprefix $(_self_dir_openshift_deps), \
 	../../lib/golang.mk \
 )
 

--- a/make/targets/openshift/imagebuilder.mk
+++ b/make/targets/openshift/imagebuilder.mk
@@ -1,4 +1,7 @@
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	../../lib/golang.mk \
+	../../lib/tmp.mk \
+)
 
 IMAGEBUILDER_VERSION ?=1.2.1
 
@@ -20,11 +23,3 @@ else
 	echo "Warning: Installed imagebuilder version $(_imagebuilder_installed_version) does not match expected version $(IMAGEBUILDER_VERSION)."
 endif
 .PHONY: ensure-imagebuilder
-
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
-	../../lib/golang.mk \
-	../../lib/tmp.mk \
-)

--- a/make/targets/openshift/operator/profile-manifests.mk
+++ b/make/targets/openshift/operator/profile-manifests.mk
@@ -1,4 +1,8 @@
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	../../../lib/tmp.mk \
+	../yq.mk \
+	../yaml-patch.mk \
+)
 
 # Merge yaml patch using mikefarah/yq
 # $1 - patch file
@@ -74,13 +78,3 @@ endef
 define add-profile-manifests
 $(eval $(call add-profile-manifests-internal,$(1),$(2),$(3)))
 endef
-
-
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
-	../../../lib/tmp.mk \
-	../yq.mk \
-	../yaml-patch.mk \
-)

--- a/make/targets/openshift/operator/telepresence.mk
+++ b/make/targets/openshift/operator/telepresence.mk
@@ -1,5 +1,4 @@
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
-scripts_dir :=$(shell realpath $(self_dir)../../../../scripts)
+scripts_dir :=$(shell realpath $(dir $(lastword $(MAKEFILE_LIST)))../../../../scripts)
 
 telepresence:
 	$(info Running operator locally against a remote cluster using telepresence (https://telepresence.io))

--- a/make/targets/openshift/rpm.mk
+++ b/make/targets/openshift/rpm.mk
@@ -1,3 +1,7 @@
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	../../lib/golang.mk \
+)
+
 RPM_OUTPUT_DIR ?=_output
 RPM_TOPDIR ?=$(abspath ./)
 RPM_BUILDDIR ?=$(RPM_TOPDIR)
@@ -32,10 +36,3 @@ clean-rpms:
 .PHONY: clean-rpms
 
 clean: clean-rpms
-
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
-	../../lib/golang.mk \
-)

--- a/make/targets/openshift/yaml-patch.mk
+++ b/make/targets/openshift/yaml-patch.mk
@@ -1,6 +1,10 @@
 ifndef _YAML_PATCH_MK_
 _YAML_PATCH_MK_ := defined
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	../../lib/golang.mk \
+	../../lib/tmp.mk \
+)
 
 YAML_PATCH ?=$(PERMANENT_TMP_GOPATH)/bin/yaml-patch
 yaml_patch_dir :=$(dir $(YAML_PATCH))
@@ -24,12 +28,4 @@ clean-yaml-patch:
 
 clean: clean-yaml-patch
 
-
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
-	../../lib/golang.mk \
-	../../lib/tmp.mk \
-)
 endif

--- a/make/targets/openshift/yq.mk
+++ b/make/targets/openshift/yq.mk
@@ -1,6 +1,10 @@
 ifndef _YQ_MK_
 _YQ_MK_ := defined
-self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	../../lib/golang.mk \
+	../../lib/tmp.mk \
+)
 
 YQ ?=$(PERMANENT_TMP_GOPATH)/bin/yq
 yq_dir :=$(dir $(YQ))
@@ -24,12 +28,4 @@ clean-yq:
 
 clean: clean-yq
 
-
-# We need to be careful to expand all the paths before any include is done
-# or self_dir could be modified for the next include by the included file.
-# Also doing this at the end of the file allows us to use self_dir before it could be modified.
-include $(addprefix $(self_dir), \
-	../../lib/golang.mk \
-	../../lib/tmp.mk \
-)
 endif

